src/index.html

```diff
@@ -11,7 +11,110 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="block"> Surname:
+          <input
+          type="text"
+          name="text"
+          minlength="3"
+          maxlength="20"
+          required
+          >
+        </label>
+        <label class="persInform"> Name:
+          <input
+          type="text"
+          name="text"
+          required
+          >
+        </label>
+        <label class="persInform"> How old are You?
+          <input
+          type="number"
+          min="1"
+          max="100"
+          name="old"
+          value="12"
+          >
+        </label>
+        <label class="persInform"> Full date of birth
+          <input type="date" name="date">
+        </label>
+        <label class="persInform"> I accept the term of the agreement
+          <input type="checkbox" name="checkbox">
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldSets">
+        <legend>Registration</legend>
+        <label class="block"> E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          >
+        </label>
+        <label class="persInform"> Password:
+          <input type="password" name="password">
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldSets">
+        <legend>An interesting fact about you!</legend>
+        <label class="block"> Do your love cats?
+          <input
+          type="radio"
+          name="radio"
+          value="Yes"
+          > Yes
```

Comment:
```suggestion
         <label>
            <input
              type="radio"
              name="radio"
              value="Yes"
            > 
            Yes
         <label>
```

better to wrap it into label